### ISSUE TITLE
Description is empty for Scenario Outline

### ISSUE THREAD

**Issue by KarinaKedzia** [behave/behave#850]
I wanted to access description of Scenarios but the problem is that in case of Scenario Outline description is empty - [ ]. In case of normal Scenario this problem does not occur.

Example of feature file where I tested mentioned behavior:
```
Feature: Some feature to test

  Scenario: TC_001 Some scenario
  Description
    Given Something that is given
    When One happens
    Then Happens something else

  Scenario Outline: Some other scenario
    Description
    Given Something that is given <data_1>
    When One happens
    Then Happens something else
    Examples:
    |data_1|
    | A    |
```
How I access it:
```
from behave.runner import Context
import pprint

def after_scenario(context: Context, scenario):
    print('>>>>')
    pprint.pprint(scenario.description)
```
and finally output of tests:
```
Feature: Some feature to test # features/test.feature:1

 

  Scenario: TC_001 Some scenario  # features/test.feature:4
    Given Something that is given # features/steps/test.py:4
    When One happens              # features/steps/test.py:9
    Then Happens something else   # features/steps/test.py:14
>>>>
['Description']

 

  Scenario Outline: Some other scenario -- @1.1   # features/test.feature:19
    Given Something that is given A               # features/steps/test.py:19
    When One happens                              # features/steps/test.py:9
    Then Happens something else                   # features/steps/test.py:14
>>>>
[]

 

1 feature passed, 0 failed, 0 skipped
2 scenarios passed, 0 failed, 0 skipped
6 steps passed, 0 failed, 0 skipped, 0 undefined
Took 0m0.003s
```
Is it a bug or should I access the description in other way in case of Scenario Outline?

**Comment by thunguyen19** [comment]
This is an example from the code base on how you can add Description for each Scenario
```
Feature:
          Scenario: E1

            This is a simple scenario description before the steps start.
            It explains why this scenario is important. This is included in description.

            Here another scenario description line after an empty line. This is also included in description.

                Given a step passes with "Alice"
                When a step passes with "Bob"
                Then a step passes with "Charly"
        """
```
And yes you can access description in after_scenario by doing `scenario.description`.
Hope this helps.

**Comment by KarinaKedzia** [comment]
Hey, @thunguyen19 

Thank for your attention. The issue is with Scenario Outline, not the Scenario. 
Could you confirm that it works with Scenario Outline not only Scenario ?

**Comment by thunguyen19** [comment]
Hi @KarinaKedzia 
Looks like the current code base doesn't support to access description in Scenario Outline by doing `scenario.description`. However, if you take a look at https://github.com/behave/behave/blob/master/tests/unit/test_parser.py#L1285, can also access description by doing `feature.scenario[index].description` in your `after_feature`. I know this may not be ideal.

Maybe this could be a bug or an enhancement for Scenario Outline?

**Comment by jenisys** [comment]
DUPLICATES: #725 (ALREADY-FIXED)

The current HEAD of the repository contains the fix:

```sh
$ git log behave/model.py
...
commit 0ae5e867f73ba50744531024e3bad79979f1dbaa
Author: jenisys <jenisys@users.noreply.github.com>
Date:   Mon Mar 11 22:37:04 2019 +0100

    FIXES #725: ScenarioOutlineBuilder was not copying description to created Scenario.
```

**Comment by jenisys** [comment]
.
